test/cpp/lazy/test_lazy_ops.cpp

```diff
@@ -7311,6 +7311,26 @@ TEST_F(LazyOpsTest, TestNarrowCopy) {
   }
 }
 
+TEST_F(LazyOpsTest, NonZeroNarrowCopy) {
+  FLAGS_ltc_enable_symbolic_shapes = true;
+  auto x_cpu = torch::rand({10});
+  auto y_cpu = torch::nonzero(x_cpu);
+  int64_t y0_cpu_size = y_cpu.sizes()[0];
+  auto b_cpu =
```

Comment:
i'm confused why you have a separate x_cpu and b_cpu tensor.

also, is torch::rand a good way to exercise nonzero?  you'll get 10 nonzero elements some really high percent of the time